commands/PM/modules.b.inc

```diff
@@ -438,3 +450,108 @@ function _uninstall_project($project) {
     b_set_message("Failed to uninstall module " . $module['name'], 'error');
   }
 }
+
+/**
+ * pm-info callback. Provide info about extention.
+ *
+ * @param array $arguments
+ * @param array $options
+ */
+function modules_b_info_callback($arguments, $options) {
+  if (!isset($arguments['modules'])) {
+    b_set_message('Please provide project name(s)', 'error');
+    return;
+  }
+  foreach($arguments['modules'] as $project ) {
+    _info_project($project);
+  }
+}
+
+/**
+ * Internal function to provide detailed information about module or theme.
+ *
+ * @param string $project
+ *  The machine name of the project about which info is required.
+ */
+function _info_project($project) {
+  $query = db_select('system', 's')
+      ->fields('s');
```

Comment:
Not entirely sure whether this way of line wrapping is correct according to coding standards; not sure about the indentation either.

Reply:
It's ok, we have coding standards checks in (`b`) core now :grin: